### PR TITLE
impl for golang package structure

### LINKED ISSUE #11: github action for go ci/cd

### PR DESCRIPTION
**Pull request by SamYuan1990**
resolve #11

Signed-off-by: Sam Yuan <yy19902439@126.com>

### CODE REVIEW

File: go/Makefile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.PHONY: unit-test
+unit-test:
```
Comment (davidkhala) [inline go/Makefile:2]: 我们有CI的描述文件和文档的话，是不是可以不写make文件？
不过go语言可能没有npm run <scripts> 这样的支持，make是起到这类似的作用么？

Comment (SamYuan1990) [reply go/Makefile:2]: 嗯 make 起到 npm run的支持作用。

**Comment by SamYuan1990** [comment]
@davidkhala could you please check why github action is not enabled?

**Comment by davidkhala** [comment]
> @davidkhala could you please check why github action is not enabled?

Not yet, I haven't add `pull_request` as trigger on any workflow. So it can only be triggered if the pull requester is another branch under `Hyperledger-TWGC/fabric-admin-sdk`

**Comment by SamYuan1990** [comment]
> > @davidkhala could you please check why github action is not enabled?
> 
> Not yet, I haven't add `pull_request` as trigger on any workflow. So it can only be triggered if the pull requester is another branch under `Hyperledger-TWGC/fabric-admin-sdk`

so... how we verify github action?

**Comment by SamYuan1990** [comment]
merge it for now to continue build up CI structure.

**Pull request merged**